runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -5182,9 +5187,10 @@ bool CPUThrottleEnabled(TR::CompilationInfo *compInfo, uint64_t crtTime)
       }
    return true;
    }
-
-/// Sums up CPU utilization of all compilation thread and write this
-/// value in the compilation info (or -1 in case of error)
+/**
```

Comment:
May I ask you to add a space before the `/**` so that it lines up with asterisks on the following lines?